src/test/java/com/puppycrawl/tools/checkstyle/checks/coding/IllegalTokenCheckTest.java

```diff
@@ -79,22 +79,21 @@ public void testCommentContentToken()
         final String[] expected = {
             "1:3: " + getCheckMessage(MSG_KEY,
                         JavadocUtil.escapeAllControlChars(
-                            " // violation\\n"
-                            + "IllegalToken\\n" + "tokens = COMMENT_CONTENT"
+                        "IllegalToken\\n" + "tokens = COMMENT_CONTENT"
                             + lineSeparator
                             + lineSeparator
                             + lineSeparator)),
             "10:3: " + getCheckMessage(MSG_KEY,
                         JavadocUtil.escapeAllControlChars(
-                            "* // violation" + lineSeparator
```

Comment:
Line numbers should be updated .
"10:3: " is line number and column number

Reply:
I have used the spaces between lines to add violation without altering line numbers used in actual tests. But, still the failure persists. I am not sure of what I am missing. While, I have spent hours and replicating the same code changes for InputIllegalTokens5.java, which doesn't flag any such failure. To me, either it is something silly i have been doing or, some other issue unknown to me.